Header header
active_object_msg[] object
